.github/workflows/release.yml

```diff
@@ -15,19 +15,22 @@ jobs:
     env:
       DOCKER_REGISTRY: docker.elastic.co
       DOCKER_SECRET: secret/observability-team/ci/docker-registry/prod
-      DOCKER_IMAGE_NAME: apm-attacher
+      DOCKER_IMAGE_NAME: observability/apm-attacher
```

Comment:
I'm a little lost on how this gets used in a deploy. Is adding `observability/` into the pathname here supported elsewhere?

Reply:
i see it is